src/containers/shared/css/tooltip.scss

```diff
@@ -1,5 +1,17 @@
 @import './variables.scss';
 
+.nft {
+  position: absolute;
+  z-index: 1000;
```

Comment:
suspicious 🤔 . why did you need this?

Reply:
yeah perhaps I shouldve done this. i added this because the new tooltip style is an entire rework of the current one, thus why it needs complete new styles. 

Should i adhere to the current tooltip style and just leave out the new styles for now? and then perhaps change the tooltips when the rework comes